tests/utests/schema/test_parser_yang.c

```diff
@@ -20,14 +20,12 @@
 #include <stdio.h>
 #include <string.h>
 
-#include "../../src/common.h"
-#include "../../src/tree_schema.h"
-#include "../../src/tree_schema_internal.h"
+#include "../../../src/common.h"
+#include "../../../src/tree_schema.h"
+#include "../../../src/tree_schema_internal.h"
```

Comment:
It may be worth it to generate one config header for all the tests (there may already be one, I have not checked) and easily put these (absolute) paths there with any other needed paths or defines. It can then be included in the tests without worrying in what directory they are or even move them around without breaking anything.

But there still would be a problem of setting the path to that one config header, now that I think about it, so it may be easier to simply change these paths to `"common.h"` and so on and use `cmake` to add include directory `src` only for the tests. Should achieve the same thing.

Reply:
maybe, I didn't want to touch this includes right now because many of them will not be necessary since we are moving from internal functions to calling API functions in tests. I think that, if it will be possible, this part can be redesigned as soon as the test functions are moved into the new joined form. I expect that some simplification then.

Reply:
Right, makes sense.